### ISSUE TITLE
--lineart unicode not working

### ISSUE THREAD

**Issue by sje30** [insanum/gcalcli#629]
Hi, 

Thanks for a great package.

`--lineart unicode` when generating calw generates the following error (fancy and ascii work as required).

```
$ gcalcli  --lineart unicode calw
Traceback (most recent call last):
  File "/sbin/gcalcli", line 33, in <module>
    sys.exit(load_entry_point('gcalcli==4.3.0', 'console_scripts', 'gcalcli')())
  File "/usr/lib/python3.10/site-packages/gcalcli/cli.py", line 167, in main
    gcal.CalQuery(
  File "/usr/lib/python3.10/site-packages/gcalcli/gcal.py", line 1282, in CalQuery
    self._GraphEvents(cmd, start, count, event_list)
  File "/usr/lib/python3.10/site-packages/gcalcli/gcal.py", line 487, in _GraphEvents
    self.printer.msg(week_top + '\n', color_border)
TypeError: can't concat str to bytes
```

**Comment by Aleuck** [comment]
Exact same error in two diferent systems (OpenSuse Tumbleweed and Raspberry Pi OS).

**Comment by dbarnett** [comment]
Fixed via #582.
